lib/plugins/aws/package/compile/events/apiGateway/lib/validate.js

```diff
@@ -42,6 +42,8 @@ module.exports = {
     const events = [];
     const corsPreflight = {};
 
+    this.validateRestApiRootResourceId();
```

Comment:
Let's put this logic inline, as this logic is not extensive and there's no re-usability use case, I believe there's no point to delegate it to dedicated method

Reply:
yes. makes sense